docs/FAQ.md

```diff
@@ -0,0 +1,33 @@
+# FAQ
+
+> Because redux-observable usage heavily depends on RxJS, most of your questions will likely be RxJS questions not specific to redux-observable itself. We strongly encourage you to seek help for these from the RxJS community at large.
+> 
+> If your redux-observable question isn't yet listed here, please [use Stack Overflow](http://stackoverflow.com/questions/tagged/redux) first. If you don't receive a response after a reasonable amount of time, create an issue ticket that includes a link to your Stack Overflow question.
+
+## Table of Contents
+
+- **General**
+  - [Does this work with RxJS 4/most.js/bacon.js?](#general-rxjs-v4)
+  - [Why were thunkservables deprecated?](#general-thunkservables-deprecated)
+
+## General
+
+<a id="general-rxjs-v4"></a>
+### Does this work with RxJS 4/most.js/bacon.js? 
+
+##### RxJS v4
+
+Not out of box. We'd love to provide an elegant for solution for this soon, ideally [RxJS v4 would add interop with `Symbol.observable`](https://github.com/Reactive-Extensions/RxJS/issues/1225).
```

Comment:
@istarkov can you create a ticket with your full thoughts on how this would work?